pycqed/measurement/quantum_experiment.py

```diff
@@ -542,15 +542,15 @@ def _configure_mc(self, MC=None):
 
         # Configure detector function
         # FIXME: this should be extended to meas_objs that are not qubits
-        df = mqm.get_multiplexed_readout_detector_functions(
+        self.df = mqm.get_multiplexed_readout_detector_functions(
             self.meas_objs, **self.df_kwargs)[self.df_name]
-        self.MC.set_detector_function(df)
+        self.MC.set_detector_function(self.df)
         if self.dev is not None:
             meas_obj_value_names_map = self.dev.get_meas_obj_value_names_map(
-                self.meas_objs, df)
+                self.meas_objs, self.df)
         else:
             meas_obj_value_names_map = mqm.get_meas_obj_value_names_map(
-                self.meas_objs, df)
+                self.meas_objs, self.df)
```

Comment:
Do we want to keep these changes? They do not seem to be necessary anymore for the functionality of the PR, but it might be useful to have the df accessible from the notebook for debugging. What do you think? I do not have a strong opinion here.

Reply:
Yes I very much like to keep these. I often want to access this attribute. It is a bug that the self.df is assigned to None in the init but then never reassigned to something useful.